core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -18,8 +18,9 @@
 package org.apache.spark
 
 import scala.collection.JavaConverters._
```

Comment:
Sorry for the demand, but can we remove this implicit conversion? I'm worried this coupled with the concurrent map will make this file more prone to bugs in future development. Also because this adds a bunch of methods that are not as clear as the Java methods (like contains in the Scala version of the Map).